docs/cli_commands.md

```diff
@@ -175,7 +175,7 @@ Creates a keymap.json from a keymap.c.
 **Usage**:
 
 ```
-qmk c2json [--no-cpp] [-o OUTPUT] filename
+qmk c2json [-h] -km KEYMAP -kb KEYBOARD [-q] [-o OUTPUT] [--no-cpp] filename
```

Comment:
```suggestion
qmk c2json -km KEYMAP -kb KEYBOARD [-q] [-o OUTPUT] [--no-cpp] filename
```

Reply:
Solved in commit [874ac88](https://github.com/qmk/qmk_firmware/commit/874ac88dd5ef9e090045d6e6d458753475ee8dd6)